### PR TITLE
New a11y demo

### PR DESCRIPTION
**Pull request by zorkow**
This is the a11y demo using a flag to switch off collapse functionality. It might still have some timing issues, although I never came across any.
@pkra can you have a look if that works for you?

### CODE REVIEW

**Comment by pkra** [comment]
Thanks for fixing this. I do see timinig issues caused by setting up the options in https://github.com/mathjax/MathJax-RespEq/blob/new_a11y_demo/Semantics-Lab/Semantic-Lab.js#L236-L239. Will try to narrow it down.

**Comment by pkra** [comment]
I don't actually understand how https://github.com/mathjax/MathJax-RespEq/blob/new_a11y_demo/Semantics-Lab/Semantic-Lab.js#L144-L146 is supposed to work but replacing it with `MathJax.Hub.Queue(MathJax.Extension.Explorer.setExplorerOption(key, value));` seems to solve the issue. I'm guessing `Semantic-Lab` should wait for `Assistive-Explorer` to be safe but it's not a proper extension anyway right now, correct?

**Comment by zorkow** [comment]
#47 Should take care of problems with setting options.

**Comment by zorkow** [comment]
Damn, forgot to rebase before pushing. It now includes #47

**Comment by zorkow** [comment]
Decided to merge all pending PR into this for ease of testing.

File: Semantics-Lab/walker/index.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>MathJax Accessibility Tool</title>
+  <script type="text/x-mathjax-config">
+    MathJax.Hub.Config({ menuSettings: {CHTMLpreview: false}, MMLorHTML: {MSIE: "HTML"},  "AssistiveMML.js": {disabled: true} });
```
Comment (pkra) [inline Semantics-Lab/walker/index.html]: This one

**Comment by pkra** [comment]
LGTM

**Comment by zorkow** [comment]
Unless there are any objections, I'll merge.

**Pull request merged**